src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data.mdx

```diff
@@ -10,10 +10,13 @@ metaDescription: 'New Relic''s Kubernetes integration: How to install and activa
 redirects:
   - /docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data
   - /docs/integrations/kubernetes-integration/understand-use-data/understand-use-data
+  - /docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#services
 ---
 
 import kubernetesDashboard from 'images/kubernetes_screenshot-full_dashboard.png'
 
+After having installed our Kubernetes integration, you can check our detailed [Find and use your Kubernetes data](/docs/integrations/kubernetes-integration/understand-use-data/) pages.
```

Comment:
@nbaenam consider streamlining this: After installing our Kubernetes integration....

Also, that link failed for me, and I wasn't sure where it would go since I seemed to be on the best page to explain the Kubernetes UI, no?

Reply:
@rhetoric101, I've checked that paragraph and I don't know why is there... The original page doesn't include it and I don't remember reviewing this page because I've only worked on the installation chapter. I'm going to remove it.